Odata-docs/client/v7/get-response-content.md

```diff
@@ -8,27 +8,74 @@ ms.date: 7/1/2019
 ms.topic: article
  
 ---
-# Data modification response
+# Get Response Content
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
 
+In the previous section, we looked at how to carry out basic CRUD operations using the OData Client. In this section, we'll go ahead and drill a little deeper into how to handle the response object for each of these operations. We'll start with data modification responses for create, update and delete operations. We'll also look at query operations for read operations.
```

Comment:
Quoting previous section here presumes that the user is covering the content page by page. I believe the better way to refer to another page is by using a link. If someone without being aware of the "previous section" reference reorganizes the content, that will invalidate the reference. Plus you can't really refer to this as a "section", it's a "page" or "document"
E.g.
On [this](/odata/client/link) page, we looked at how to carry out basic CRUD operations using the OData Client. In this page, we'll go ahead and drill a little deeper into how to handle the response object for each of these operations.

or

In this page, we'll look at how to handle the responses from [basic CRUD operations](/odata/client/link).

Reply:
#Resolved